views/dashboard/html-export.php

```diff
@@ -38,6 +38,12 @@
 					<span>This will <strong>only</strong> attempt to find and fix term/taxonomy relationships.</span>
 				</td>
 			</tr>
+			<tr valign="top">
+				<th scope="row">Sync Post Since Date</th>
```

Comment:
We should probably specify the datetime as "EST" since we dont use unix timestamp or GMT time.

Reply:
Good call, will add that.